prow/cluster/jobs/istio/operator/istio.operator.master.yaml

```diff
@@ -32,10 +32,39 @@ presubmits:
       - <<: *istio_container
         command:
         - entrypoint
-        - prow/istio-lint.sh
+        - prow/lint.sh
       nodeSelector:
         testing: test-pool
 
+  - name: operator-build-master
+    <<: *job_template
```

Comment:
FYI: I recommend writing plain yaml configs (generating via additional tooling when necessary) and avoiding the use of complicated yaml functionality like anchors -- this complexity adds friction that decreases the number of people willing to modify jobs.